[2023-06-13T06:59:01.565+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: add_entry_dag.add_entry_to_db_task scheduled__2023-06-13T06:58:00+00:00 [queued]>
[2023-06-13T06:59:01.571+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: add_entry_dag.add_entry_to_db_task scheduled__2023-06-13T06:58:00+00:00 [queued]>
[2023-06-13T06:59:01.572+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-13T06:59:01.572+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-06-13T06:59:01.572+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-13T06:59:01.580+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): add_entry_to_db_task> on 2023-06-13 06:58:00+00:00
[2023-06-13T06:59:01.584+0000] {standard_task_runner.py:55} INFO - Started process 499 to run task
[2023-06-13T06:59:01.587+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'add_entry_dag', 'add_entry_to_db_task', 'scheduled__2023-06-13T06:58:00+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/AssignmentDag.py', '--cfg-path', '/tmp/tmpgowvdavx']
[2023-06-13T06:59:01.588+0000] {standard_task_runner.py:83} INFO - Job 69: Subtask add_entry_to_db_task
[2023-06-13T06:59:01.641+0000] {task_command.py:389} INFO - Running <TaskInstance: add_entry_dag.add_entry_to_db_task scheduled__2023-06-13T06:58:00+00:00 [running]> on host 1840d8ec0686
[2023-06-13T06:59:01.693+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=add_entry_dag
AIRFLOW_CTX_TASK_ID=add_entry_to_db_task
AIRFLOW_CTX_EXECUTION_DATE=2023-06-13T06:58:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-13T06:58:00+00:00
[2023-06-13T06:59:01.702+0000] {base.py:73} INFO - Using connection ID 'postgres_connection' for task execution.
[2023-06-13T06:59:01.707+0000] {sql.py:364} INFO - Running statement: INSERT INTO assignment_table (execution_time) VALUES ('2023-06-13 06:59:01.694799'), parameters: None
[2023-06-13T06:59:01.708+0000] {sql.py:373} INFO - Rows affected: 1
[2023-06-13T06:59:01.709+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-13T06:59:01.718+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=add_entry_dag, task_id=add_entry_to_db_task, execution_date=20230613T065800, start_date=20230613T065901, end_date=20230613T065901
[2023-06-13T06:59:01.761+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-13T06:59:01.775+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
